<?xml version="1.0"?>
<robot name="chassis_with_single_arm">
  <virtual_joint name="world_joint" type="planar" parent_frame="odom_link" child_link="chassis_center_link"/>
  <group name="manipulator">
    <chain base_link="left_arm_base_link" tip_link="left_gripper_link"/>
  </group>
  <group name="mobile_base">
    <joint name="world_joint"/>
  </group>
  <group name="whole_body">
    <group name="mobile_base"/>
    <group name="manipulator"/>
  </group>
</robot>
